Release/include/cpprest/oauth2.h

```diff
@@ -479,6 +481,17 @@ class oauth2_config
     /// </summary>
     void set_user_agent(utility::string_t user_agent) { m_user_agent = std::move(user_agent); }
 
+    /// <summary>
+    /// Set the certificate chain callback to be used by the http client.
+    /// </summary>
+    void set_user_certificate_chain_callback(const CertificateChainCallBackFunction& callback) { m_certificate_chain_callback = callback; }
+
+    /// <summary>
+    /// Get the cert chain callback.
+    /// </summary>
+    /// <returns>A reference to cert chain callback user by the client.</returns>
+    const CertificateChainCallBackFunction& user_certificate_chain_callback() { return m_certificate_chain_callback; }
```

Comment:
Not sure about returning const ref to this, what does it achieve in this case for std:function?